### DISCUSSION TITLE
Data Driven Testing: Any example on how to incorporate processing each csv row to a test function

### DISCUSSION THREAD

**Discussion by narendarbhogala** [seleniumbase/SeleniumBase#2015] [Q&A] [answered]
Can anyone provide any example on how they have incorporated Data Driven testing with seleniumBase. 
If any one has implemented processing test function for each record of csv read will be extremely helpful.

**Comment by mdmintz** [comment, marked as answer]
For a file-parsing example, see: https://github.com/seleniumbase/SeleniumBase/blob/master/examples/boilerplates/samples/file_parsing/parse_files.py

For run commands to store in a CSV, you can use ``pytest --co -q``, and then collect them into a file, and then parse them.
